<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppFormQtCpp</class>
 <widget class="QMainWindow" name="CIhmAppFormQtCpp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2025</width>
    <height>1607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formation Qt C++ (PhA-2017)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbCapteurs">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Capteurs I2C</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>171</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Humidité :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>61</y>
       <width>246</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Température : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>51</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>51</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>181</width>
       <height>103</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlI2c">
      <item>
       <widget class="QLineEdit" name="leTempI2c">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leHumI2c">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>161</width>
       <height>103</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="leSeuilTempI2c"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leSeuilHumI2c"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>200</y>
       <width>301</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Seuil de temp :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Seuil d'humidité :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbStartStop">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>491</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Start acquisitions</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbCommander">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>671</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Commander la LED</string>
    </property>
    <widget class="QPushButton" name="pbOnOffLed">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>199</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Allumer</string>
     </property>
    </widget>
    <widget class="QLabel" name="laNoir">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>noir.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="laRouge">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>rouge.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>laRouge</zorder>
    <zorder>pbOnOffLed</zorder>
    <zorder>laNoir</zorder>
   </widget>
   <widget class="QTextEdit" name="teTexte">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>1241</width>
      <height>821</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Capteur SPI</string>
    </property>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>251</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Température :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>261</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Seuil de temp :</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlSpi">
      <item>
       <widget class="QLineEdit" name="leTempSpi">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSeuilTempSpi"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>51</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>110</y>
       <width>51</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2025</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>1500</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Initialisation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="gbIntervalle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>671</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>Intervalles</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>170</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Capteurs :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>302</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Base de données :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>299</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Serveur (réseau) :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>115</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Ecran :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>17</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>17</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>109</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>111</width>
        <height>257</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vlInter">
       <item>
        <widget class="QLineEdit" name="leInterCapt">
         <property name="inputMask">
          <string>99; </string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leInterBdd">
         <property name="inputMask">
          <string>99; </string>
         </property>
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leInterServ">
         <property name="inputMask">
          <string>99; </string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leInterEcran">
         <property name="inputMask">
          <string>99; </string>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>671</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Paramètres liaison série</string>
     </property>
     <widget class="QComboBox" name="cbPorts">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>221</width>
        <height>46</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>319</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Choix de la liaison :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbVitesse">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>111</y>
        <width>221</width>
        <height>46</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbParite">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>162</y>
        <width>221</width>
        <height>47</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Sans parité</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Impaire</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbBits">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>214</y>
        <width>71</width>
        <height>46</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbStop">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>265</y>
        <width>71</width>
        <height>46</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>337</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Vitesse :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Parité :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Nbre de bits/car :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Nbre de bit de stop :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbServeur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>790</y>
       <width>681</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Paramètres serveur</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>193</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse IP :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>88</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>281</width>
        <height>103</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vlServer">
       <item>
        <widget class="QLineEdit" name="leAdrServ">
         <property name="inputMask">
          <string>999.999.999.999; </string>
         </property>
         <property name="text">
          <string>192.168.2.100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePortServ">
         <property name="text">
          <string>5555</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbSgbd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>990</y>
       <width>681</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Paramètres SGBD</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>193</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse IP :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>276</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom de la base :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>274</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom utilisateur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>238</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>348</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vlSgbd">
       <item>
        <widget class="QLineEdit" name="leAdrSgbd">
         <property name="inputMask">
          <string>999.999.999.999; </string>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leNomBdd">
         <property name="text">
          <string>bddFormation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leUserSgbd">
         <property name="text">
          <string>root</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePassSgbd">
         <property name="text">
          <string>admin</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
